scripts/tools/CleanUpOrphanFiles.php

```diff
@@ -0,0 +1,239 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\generis\scripts\tools;
+
+use common_report_Report as Report;
+use core_kernel_classes_Resource;
+use oat\generis\model\fileReference\ResourceFileSerializer;
+use oat\generis\model\GenerisRdf;
+use oat\generis\model\OntologyAwareTrait;
+use oat\generis\model\resource\ResourceCollection;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\filesystem\FileSystemHandler;
+
+/**
+ * sudo -u www-data php index.php 'oat\generis\scripts\tools\CleanUpOrphanFiles'
+ */
+class CleanUpOrphanFiles extends ScriptAction
+{
+
+    use OntologyAwareTrait;
+
+    private $wetRun = false;
+    private $verbose = false;
+    private $removedCount = 0;
+    private $redundantCount = 0;
+    private $affectedCount = 0;
+    private $errorsCount = 0;
+    private $report;
+    private $markedTobeRemoved =[];
+
+    protected function provideOptions()
```

Comment:
Some DocBlocks are missing

Reply:
We are going to remove this script anyway. Is there a point to add docblocks now?